templates/base/packages/nextjs/tsconfig.json.template.mjs

```diff
@@ -9,7 +12,7 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "${moduleResolution}",
```

Comment:
Umm seems too specific, maybe we should update the main SE-2 repo with "moduleResolution: esnext" by default? 

I just tried `npx create-next-app@latest` and its default `tsconfig` for latest next scaffold app is `"module": "esnext",`

Reply:
> Umm seems too specific, maybe we should update the main SE-2 repo with "moduleResolution: esnext" by default?

There is no option `esnext` for `moduleResolution`, have you meant `NodeNext`?

I thought about it, but for me only "Bundler" works for both extension and se-2. With "NodeNext" I'm getting an errors for every `wagmi`/`abitype` imports. Example: 
<img width="1055" alt="image" src="https://github.com/scaffold-eth/create-eth/assets/25638585/b318037e-3c38-43ef-97a7-576bb2a7a60d">

It asks for changing to "type": "module" on package.json, after it I'm starting to get other nextjs import errors etc etc. I didn't find a good solution for now, but I think we need to create an issue to tackle with it. As I remember we need to add support for latest nodejs version, probably we can try to fix both at the same time (but not sure about it)

Reply:
Ohh my bad, I just confused b/w "module" and "moduleResolution". I mean to say maybe its good idea to switch to `"moduleResolution: "bundler"` ("node" => "bundler"). Since `npx create-next-app@latest` the `tsConfig`  their uses `moduleResolution: "bundler"` as default and also for tackling https://github.com/scaffold-eth/create-eth-extensions/pull/11 , we need it to be `moduleResolution: "bundler"` so maybe its good idea we keep it as "bundler" by default in main SE-2 as well? So that we will be future proof("bundler" seems new & more prefered now?), also people building new extension don't need pass that field in `tsconfig.template.mjs` file and also decreases the `.temlate.mjs` for us too.

Reply:
[841bc48](https://github.com/scaffold-eth/create-eth/pull/74/commits/841bc48ce89282765265b7f702a0ce509825d26e) finally removed tsconfig template